081-unregistration-of-KRaft-nodes.md

```diff
@@ -0,0 +1,117 @@
+# Unregistration of KRaft nodes
```

Comment:
List in readme?

Reply:
I do that only at the end once I know the final link, number, filename etc.